src/foundry/foundry.js/clientDocumentMixin.d.ts

```diff
@@ -283,7 +283,7 @@ export declare class ClientDocumentMixin<T extends foundry.abstract.Document<any
       | ConstructorDataType<InstanceType<T>['data']>
       | (ConstructorDataType<InstanceType<T>['data']> & Record<string, unknown>)
     >,
-    options?: Dialog.Options
+    context?: DocumentModificationContext & Dialog.Options
```

Comment:
```suggestion
    context?: DocumentModificationContext & Partial<Dialog.Options>
```
Also, only the `parent` and `pack` properties from `DocumentModificationContext` are actually considered, all other properties are ignored. So maybe just pick those 2 properties...?

Reply:
You are right, I missed the `extends` on `DocumentModificationContext`.